app/controllers/spree/admin/resource_controller.rb

```diff
@@ -251,6 +251,10 @@ def object_url(object = nil, options = {})
         end
       end
 
+      def permitted_resource_params
+        params.require(object_name).permit!
```

Comment:
As you mentioned before, we could have this method return an error instead of permitting everything. It's a bold move, but I think it might be the right choice...

Reply:
ok, I agree, I'll do that in this PR

Reply:
done, it's now raising am error.